app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -5,90 +5,118 @@ window.GOVUK.Modules = window.GOVUK.Modules || {}
-        var checkbox = e.target
-        var isNested = $(checkbox).closest('.govuk-checkboxes--nested')
-        var hasNested = $('.govuk-checkboxes--nested[data-parent=' + checkbox.id + ']')
-
-        if (hasNested.length) {
-          _this.toggleNestedCheckboxes(hasNested, checkbox)
-        } else if (isNested.length) {
-          _this.toggleParentCheckbox(isNested, checkbox)
-        }
-      })
-
-      $(scope).on('change', 'input[type=checkbox]', function (e) {
-        if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
-          // where checkboxes are manipulated externally in finders, suppressAnalytics
-          // is passed to prevent duplicate GA events
-          if (typeof e.suppressAnalytics === 'undefined' || e.suppressAnalytics !== true) {
-            var $checkbox = $(e.target)
-            var category = $checkbox.data('track-category')
-            if (typeof category !== 'undefined') {
-              var isChecked = $checkbox.is(':checked')
-              var uncheckTrackCategory = $checkbox.data('uncheck-track-category')
-              if (!isChecked && typeof uncheckTrackCategory !== 'undefined') {
-                category = uncheckTrackCategory
-              }
-              var action = $checkbox.data('track-action')
-              var options = $checkbox.data('track-options')
-              if (typeof options !== 'object' || options === null) {
-                options = {}
-              }
-              options.value = $checkbox.data('track-value')
-              options.label = $checkbox.data('track-label')
-              window.GOVUK.analytics.trackEvent(category, action, options)
-            }
+  function GovukCheckboxes () { }
+
+  GovukCheckboxes.prototype.start = function ($module) {
+    this.$module = $module[0]
+    this.$checkboxes = this.$module.querySelectorAll('input[type=checkbox]')
+    this.$nestedCheckboxes = this.$module.querySelectorAll('[data-nested=true] input[type=checkbox]')
+    this.$exclusiveCheckboxes = this.$module.querySelectorAll('[data-exclusive=true] input[type=checkbox]')
+
+    this.applyAriaControlsAttributes(this.$module)
+
+    for (var i = 0; i < this.$checkboxes.length; i++) {
+      this.$checkboxes[i].addEventListener('change', this.handleCheckboxChange)
+    }
+
+    for (i = 0; i < this.$nestedCheckboxes.length; i++) {
+      this.$nestedCheckboxes[i].addEventListener('change', this.handleNestedCheckboxChange.bind(this))
+    }
+
+    for (i = 0; i < this.$exclusiveCheckboxes.length; i++) {
+      this.$exclusiveCheckboxes[i].addEventListener('change', this.handleExclusiveCheckboxChange)
+    }
+  }
+
+  GovukCheckboxes.prototype.handleCheckboxChange = function (event) {
+    if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
+      // where checkboxes are manipulated externally in finders, suppressAnalytics
+      // is passed to prevent duplicate GA events
```

Comment:
```suggestion
      // Where checkboxes are manipulated externally in finders, `suppressAnalytics`
      // is passed to prevent duplicate GA events.
```